### PR TITLE
Update structure of website and added a navbar to navigate pages

### PR DESCRIPTION
**Pull request by cjoshea9**
Restructure the website from scratch to now have a landing page and other linked pages in the navbar. 

https://photos.google.com/photo/AF1QipPUj_xNZzbh4yJBB4OWHSyyQdzwQR4D-_8NDKAw

### CODE REVIEW

**Comment by cjoshea9** [comment]
It was just an image of my very basic home page.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -6,19 +6,34 @@
...
+        </nav>
+      </header>
+    </div>
     <div id="content">
-      <div id="intro" background="/images/utah.jpg">  
-        <h1>Cory's Portfolio </h1>
-        <p>This is my portfolio. I am currently a sophomore at MIT studying Computer Science and a member of the varsity baseball team.</p>
+      <div id="intro">  
+        <h1 class="intro-header">Hi, I'm Cory O'Shea </h1>
+
```
Comment (Danish21) [inline portfolio/src/main/webapp/index.html:28]: Nit: Remove extra new line.
In most places you have no spacing between tags, but here you have a space. I recommend picking a spacing strategy and being consistent everywhere. 
https://google.github.io/styleguide/htmlcssguide.html#Parting_Words

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -6,19 +6,34 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body >
```
Comment (Danish21) [inline portfolio/src/main/webapp/index.html:10]: Remove empty space

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -6,19 +6,34 @@
...
+    <div>
+      <header> 
+        <nav>
+          <div>
+            <ul id="navbar-div">
+              <li><a href="resume.html">Resume</a> </li>
```
Comment (Danish21) [inline portfolio/src/main/webapp/index.html:16]: Move the nest a blocks to a new and indented line to respect the style guide:
https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

**Pull request merged**